### DISCUSSION TITLE
Relation manager in create page

### DISCUSSION THREAD

**Discussion by iranandroid** [filamentphp/filament#5112] [Help]
Hi, i'm new in laravel and filament. now i wanna add list of country( like below picture) table to create page  and user can enable, but i don't know

<img width="1265" alt="image" src="https://user-images.githubusercontent.com/20900080/204882245-67ef1ce7-4456-4ceb-82b7-4d08f6128fd6.png">

**Comment by antoinemy** [comment]
up, any one? Can we create the relationships with the RelationManagers on a CreateResource like on the EditResource or ViewResource intuitively? By creating the relation only after having created the resource?

**Reply by danharrin** [reply]
relation managers require the record to already be created. but you could use a Repeater instead for simple cases - https://filamentphp.com/docs/2.x/forms/fields#populating-automatically-from-a-relationship-2

**Reply by antoinemy** [reply]
Ok, thanks @danharrin, I wanted to make sure I wasn't missing a native Filamentphp option!

**Reply by kisexu** [reply]
Thanks, it works for me.

**Reply by franklinogf** [reply]
> relation managers require the record to already be created. but you could use a Repeater instead for simple cases - https://filamentphp.com/docs/2.x/forms/fields#populating-automatically-from-a-relationship-2

Can the repeater be used with spatie translatable plugin?
